README.md

```diff
@@ -1,82 +1,13 @@
 <br>
 <img src="https://user-images.githubusercontent.com/4295811/226700092-ffbd0c01-dba1-4880-8b77-a4d26e6228f0.svg"  width="64">
 
-# README
+# Rails on Aptible
 
 This is a variation of the [Rails](https://rubyonrails.org/) [Getting Started](https://guides.rubyonrails.org/v4.2.7/getting_started.html) example on [Aptible](https://aptible.com).
 
 The app in this repo is deployed live [here](https://app-52710.on-aptible.com/).
 
-## Prerequisites
+[![Deploy on Aptible](https://app.aptible.com/deploy-on-aptible.svg)](https://app.aptible.com/create)
 
-Please complete the following steps before moving forward:
+[Detailed docs](https://www.aptible.com/docs/ruby-quickstart)
 
-- Install [Git](https://git-scm.com/downloads)
```

Comment:
We should mention how to generate the `SECRET_KEY`.  We should also change it away from `DEVELOPMENT_SECRET_KEY`

Reply:
https://github.com/aptible/template-laravel/blob/b9dee9c15afecdb5f4c136f0d637303a6f7680f4/README.md

Reply:
`PRODUCTION_SECRET_KEY` you mean? and if so i'd have to add a corresponding change to app-ui here: https://github.com/aptible/app-ui/blob/5541ac5e4498428aea56b8afa481c80fca9b63b7/src/ui/pages/create-project-git.tsx#L562 away from `DEVELOPMENT_SECRET_KEY`, right?

Reply:
https://github.com/aptible/app-ui/pull/76/commits/b52006513cd893e059aa23727f2d3e8ddaf2d78f